src/modules/transaction/reducer.ts

```diff
@@ -0,0 +1,43 @@
+import { Network } from '@dcl/schemas'
+import { getChainIdByNetwork } from 'decentraland-dapps/dist/lib/eth'
+import {
+  transactionReducer as baseTransactionReducer,
+  TransactionReducerAction as BaseTransactionReducerAction,
+} from 'decentraland-dapps/dist/modules/transaction/reducer'
+import { TransactionStatus } from 'decentraland-dapps/dist/modules/transaction/types'
+import { InsertTransactionAction, INSERT_TRANSACTION } from './action'
+
+export type TransactionReducerAction =
+  | BaseTransactionReducerAction
+  | InsertTransactionAction
+
+const INITIAL_STATE = { data: [], loading: [], error: null }
+
+export const transactionReducer = (
```

Comment:
Workaround to extend the @decentraland/dapps transaction reducer. I don't think adding this action there should be necessary. Yet, having this wrapper does not really convince me.

Reply:
Not necessary anymore